.env

```diff
@@ -0,0 +1,6 @@
+DATABASE_NAME=
+DATABASE_USER=
+DATABASE_PASSWORD=
+DATABASE_HOST=
+DATABASE_PORT=
+MYSQL_ROOT_PASSWORD=
```

Comment:
Why are those settings not set ? Is there a database to connect already ?

Reply:
Ideally info like this should not be pushed to get repo. For example password.